meta-oe/recipes-connectivity/hiba/hiba_git.bb

```diff
@@ -17,26 +17,22 @@ PV = "1.0+git${SRCPV}"
 inherit autotools
 
 DEPENDS = "openssl openssh zlib"
-RDEPENDS:${PN} = "bash"
 
-def get_oeconf(d):
-  import glob
-  workdir = d.getVar("WORKDIR")
-  openssh_paths = glob.glob("{}/../../openssh/*/openssh-*".format(workdir))
-
-  if len(openssh_paths) != 1:
-    # Cannot fail here since this will cause failure on other recipes
-    # if openssh is not installed
-    return ""
-
-  return "--prefix=/usr/ --with-opensshdir={}".format(openssh_paths[0])
-
-EXTRA_OECONF = "${@get_oeconf(d)}"
+EXTRA_OECONF = " \
+  --with-opensshdir=${PKG_CONFIG_SYSROOT_DIR}/usr/include/ssh \
+  --prefix=/usr/ \
+"
 
 # Make sure that MACHINE_FEATURES includes hiba before continuing
 do_configure:prepend(){
-  if ${@bb.utils.contains("MACHINE_FEATURES", "hiba", "false", "true", d)} ; then
-    echo "MACHINE_FEATURES does not support hiba"
+  if ${@bb.utils.contains("DISTRO_FEATURES", "hiba", "false", "true", d)} ; then
+    echo "DISTRO_FEATURES does not support hiba. Skip the configuration"
     exit 1
   fi
 }
+
+PACKAGES =+ "${PN}-tools"
+
+FILES:${PN}-tools = "${sbindir}/hiba-ca.sh ${sbindir}/hiba-grl ${sbindir}/hiba-gen"
+
+RDEPENDS:${PN}-tools = "${PN} bash"
```

Comment:
Are these bash scripts ?  We avoid bash dependencies as much as we can and enable it absolutely when needed.

Reply:
I don't think we need that script during runtime? (maybe).

actually, we might need it. Let's see what @blunderer think.

Reply:
We do not need the bash script on the target, this is mostly for the host